test/e2e/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I'd suggest this tsconfig file, more similar to what we have in other places (eg `packages/*/tscofig.json`) but adapted for Node:

```
{
	"compilerOptions": {
		"target": "es2020",
		"lib": [ "DOM", "es2020" ],
		"baseUrl": ".",
		"module": "commonjs",
		"allowJs": false,
		"declaration": true,
		"declarationDir": "types",
		"outDir": "dist",
		"rootDir": "lib",

		"strict": true,
		"noUnusedLocals": true,
		"noUnusedParameters": true,
		"noImplicitReturns": true,
		"noFallthroughCasesInSwitch": true,

		"moduleResolution": "node",
		"esModuleInterop": true,

		"forceConsistentCasingInFileNames": true,

		"types": [ "node" ],

		"noEmitHelpers": true,
		"importHelpers": true,

		"composite": true
	},
	"include": [ "lib" ],
}
```

Reply:
Thanks. I only knew enough to get into trouble with my configuration, so I appreciate your suggestion.